besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -310,20 +312,7 @@
               + "Default is a predefined list.",
       split = ",",
       arity = "0..*")
-  void setBootnodes(final List<String> values) {
-    try {
-      bootNodes =
-          values.stream()
-              .filter(value -> !value.isEmpty())
-              .map(EnodeURL::fromString)
-              .collect(Collectors.toList());
-      DiscoveryConfiguration.assertValidBootnodes(bootNodes);
-    } catch (final IllegalArgumentException e) {
-      throw new ParameterException(commandLine, e.getMessage());
-    }
-  }
-
-  private List<EnodeURL> bootNodes = null;
+  private final List<String> bootnodes = null;
```

Comment:
Should this be camelCased?  `bootNodes`?  It was before and other places in the edited code retain the camel casing.  Whatever it chosen it should be consistent.

Reply:
Done